### ISSUE TITLE
docker_compose_v2: : Cannot parse event from line

### ISSUE THREAD

**Issue by thvd** [ansible-collections/community.docker#787]
##### SUMMARY
When using `docker_compose_v2` i get the warning: `Cannot parse event from line`.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

Command: docker_compose_v2

##### ANSIBLE VERSION

```paste below
ansible [core 2.14.9]
  config file = /home/theodoor/workspace/docker-at-home/ansible.cfg
  configured module search path = ['/home/theodoor/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3/dist-packages/ansible
  ansible collection location = /home/theodoor/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/theodoor/.local/bin/ansible
  python version = 3.11.6 (main, Oct  8 2023, 05:06:43) [GCC 13.2.0] (/usr/bin/python3)
  jinja version = 3.1.2
  libyaml = True

```

##### COLLECTION VERSION

```paste below
# /home/theodoor/.ansible/collections/ansible_collections
Collection        Version
----------------- -------
community.general 8.3.0  

# /usr/lib/python3/dist-packages/ansible_collections
Collection        Version
----------------- -------
community.general 6.6.2 
```

##### CONFIGURATION

```paste below
CALLBACKS_ENABLED(/home/theodoor/workspace/docker-at-home/ansible.cfg) = ['ansible.posix.profile_tasks']
CONFIG_FILE() = /home/theodoor/workspace/docker-at-home/ansible.cfg

```

##### STEPS TO REPRODUCE

Have an external network in your docker compose yaml config, that's not created within the same yaml project file, you will get the warning described below. (#ACTUAL RESULTS)

My docker compose yaml file contains this part: 
```yaml
networks:
  influxNetwork:
    name: influxNetwork
    driver: bridge
```

##### EXPECTED RESULTS
No warnings/errors

##### ACTUAL RESULTS

```paste below
TASK [influx : Start Influxdb] *************************************************************************************************************************************************************************************************************************
vrijdag 02 februari 2024  08:14:10 +0100 (0:00:00.995)       0:00:13.492 ****** 
[WARNING]: Cannot parse event from line: 'time="2024-02-02T08:14:10+01:00" level=warning msg="a network with name influxNetwork exists but was not created for project \\"influxdb\\".\\nSet `external: true` to use an existing network"'. Please
report this at https://github.com/ansible-collections/community.docker/issues/new?assignees=&labels=&projects=&template=bug_report.md
changed: [192.168.1.2]

```

**Comment by thvd** [comment]
My docker compose version is:
```
Docker Compose version v2.20.3
```

**Comment by felixfontein** [comment]
Hmm, I think to properly handle this I need more information from the `stderr` (or `stderr_lines`) output. Can you provide more of it than the single line `time="2024-02-02T08:14:10+01:00" level=warning msg="a network with name influxNetwork exists but was not created for project \"influxdb\".\nSet `external: true` to use an existing network"`?

**Comment by felixfontein** [comment]
(Sorry, I already wrote that on Friday, but due to the shitty internet connection I had it probably didn't make it, and I only noticed now...)

**Comment by felixfontein** [comment]
@thvd ping

**Comment by felixfontein** [comment]
@thvd ping, without further information I cannot fix this. Thanks.

**Comment by mabeett** [comment]
Hello @felixfontein , 
I've just reported #807 as thoroughly as I could as n independient ticket because I am not plenty sure if this issue is the same problem of the  situation described there.
Le me know if you need more information or the information written here.
Regards,

**Comment by jdomarin** [comment]
Hello @felixfontein ,

As you can see from attached images below, I am also facing the same warning, with a slightly different message.
Below you will find the task and the handler triggered by the task that produces the warning.
Please note it is just a warning, which does not interrupt Ansible playbook execution.
It looks like the `community.docker.docker_compose_v2_pull` module tries to parse `docker pull` output and fails to do so. 

Additional details:

Task and handler from the mentioned Ansible playbook runs in a Gitlab runner, triggered by a CI job.

- community.docker version: "3.7.0"
- docker_compose_v2_version: `2.24.5-1~ubuntu.22.04~jammy`
- docker_ce_version: "5:25.0.3"
- ansible version: 2.15

![copy_docker_compose_file_task](https://github.com/ansible-collections/community.docker/assets/29987676/8d7cc57a-ac70-4c73-8850-3c9a2f1d6107)
![pull_docker_images_handler](https://github.com/ansible-collections/community.docker/assets/29987676/5b347f7e-0418-4f37-911f-e83e2bc7abd5)
![cannot_parse_event_from_line](https://github.com/ansible-collections/community.docker/assets/29987676/880b106f-c12d-4d3b-bc8c-e3bbc2d9b2c3)

Hoping the above will help you in your investigations.
Regards.

**Comment by felixfontein** [comment]
@jdomarin thanks! That specific message was fixed in 7b554082ea343faa12517b03a9b93289bcb0597b, the fix is included in community.docker 3.8.0.

**Comment by felixfontein** [comment]
The remaining cases from this issue are fixed by #811.
